### PR TITLE
feature(HomePage): UserProfile

### PR DESCRIPTION
**Pull request by PAKovalev**
Proper structure of UserProfile stuff.

### CODE REVIEW

File: packages/app/src/scenes/home/pages/HomePage/components/templates/UserProfilePanel/UserProfilePanel.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      componentSize={{width: '390px'}}
+      headerPlaceholder
+      titleHeight
+      hasBorder={hasBorder}
+    >
+      <styled.Body data-testid="ProfileWidget-test">
+        {sessionStore.userId && userProfile && (
+          <>
+            {!isItMe ? (
+              <UserProfileView userId={userId} onClose={onClose} showUserInteractions={false} />
```
Comment (aminbksi) [inline packages/app/src/scenes/home/pages/HomePage/components/templates/UserProfilePanel/UserProfilePanel.tsx:63]: shouldn't `showUserInteractions` be `{ture}` ?

**Review by aminbksi — APPROVED** [review]

File: packages/app/src/scenes/home/stores/HomeStore/models/OnlineUsersStore/models/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from '../../UserProfileStore';
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/home/stores/HomeStore/models/OnlineUsersStore/models/index.ts:1]: this here is on purpose?

File: packages/app/src/scenes/home/pages/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -42,7 +43,20 @@ const HomePage: FC = () => {
...
-        <OnlineUsersPanel />
+        <styled.UsersContainer>
+          {onlineUsersStore.selectedUserId && (
+            <div>
+              <UserProfilePanel
+                userId={onlineUsersStore.selectedUserId}
+                onClose={() => {
+                  onlineUsersStore.unselectUser();
+                  userProfileDialog.close();
+                }}
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/home/pages/HomePage/HomePage.tsx:54]: Do we need this info in 2 stores?

**Review by dmitry-yudakov — APPROVED** [review]
approved with couple of remarks

File: packages/app/src/scenes/home/stores/HomeStore/models/OnlineUsersStore/models/index.ts (hunk 2 above)
Comment (PAKovalev) [reply packages/app/src/scenes/home/stores/HomeStore/models/OnlineUsersStore/models/index.ts:1]: ahh, I forgot to remove.

**Pull request merged**